mappings/net/minecraft/structure/HoglinStableData.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5177 net/minecraft/structure/HoglinStableData
```

Comment:
What does "stable" mean here? I also think it should be `StableHoglinData`.

Reply:
stable is like the place where you raise a horse. this is like a hoglin farm.

Reply:
It's stable as a place. The class stores the data of the hoglin stable building